.github/workflows/llm_integration.yml

```diff
@@ -8,7 +8,7 @@ on:
         required: false
         default: ''
       run_test:
-        description: 'Run only the tests you need [ft, ds, hf, aot, lora-correctness]'
+        description: 'Run only the tests you need [ft, ds, hf, aot, lora-correctness, smoothquant]'
```

Comment:
what is this for? Do we expect it to work with smoothquant?

Reply:
Removed